.NET/Microsoft.Recognizers.Text.DateTime/Chinese/Extractors/ChineseDurationExtractorConfiguration.cs

```diff
@@ -30,6 +31,21 @@ public class ChineseDurationExtractorConfiguration : ChineseBaseDateTimeExtracto
 
         private static readonly Regex HalfSuffixRegex = new Regex(DateTimeDefinitions.DurationHalfSuffixRegex, RegexFlags);
 
+        private static readonly Regex DurationUnitRegex = new Regex(DateTimeDefinitions.DurationUnitRegex, RegexFlags);
+
+        private static readonly Regex DurationConnectorRegex = new Regex(DateTimeDefinitions.DurationConnectorRegex, RegexFlags);
+
+        private static readonly Dictionary<string, string> UnitMap = DateTimeDefinitions.ParserConfigurationUnitMap.ToDictionary(k => k.Key, k => k.Value.Substring(0, 1) + k.Value.Substring(1).ToLower());
+
+        private static readonly Dictionary<string, long> UnitValueMap = DateTimeDefinitions.DurationUnitValueMap;
+
+        private readonly bool merge;
+
+        public ChineseDurationExtractorConfiguration(bool merge = true)
+        {
+            this.merge = merge;

```

Comment:
Why is this flag necessary?

Reply:
As mentioned above, extractor returns a complete text like "8天20小时“ to parser. Parser calls extractor to separate it into "8天“ and "20小时“, then recognize both value separately. At that time, it uses this flag to avoid extractor from using the MergeMultipleDuration function.

Reply:
Is this the same behaviour as in English?

Reply:
Yes. The code is in https://github.com/microsoft/Recognizers-Text/blob/master/.NET/Microsoft.Recognizers.Text.DateTime/Extractors/BaseDurationExtractor.cs#L19